torch/_inductor/lowering.py

```diff
@@ -964,9 +964,12 @@ def mkl_packed_linear(
                 b: TensorBox,
                 batch_size,
             ):
-                return TensorBox.create(
-                    ir.MKLPackedLinear.create(x, packed_w, orig_w, b, batch_size)
+                result = TensorBox.create(
+                    ir.MKLPackedLinear.create(x, packed_w, orig_w, None, batch_size)
```

Comment:
If bias is always not needed, why do we bother pass `None`? How about removing `bias` arg?

Reply:
Yes, removed it.